src/hubbardoperators.jl

```diff
@@ -239,6 +241,28 @@ function ud_num(elt::Type{<:Number}, ::Type{U1Irrep}, ::Type{SU2Irrep})
 end
 const nꜛꜜ = ud_num
 
+@doc """
+    half_ud_num([elt::Type{<:Number}], [particle_symmetry::Type{<:Sector}], [spin_symmetry::Type{<:Sector}])
+
+Return the one-body operator that counts the number of singly occupied sites.
+This is equivalent to `(nꜛ - 1/2)(nꜜ - 1/2)`, and respects the particle-hole symmetry.
+"""
```

Comment:
I guess the operator that counts the number of singly occupied sites is `1/2 - 2*half_ud_num` ?

Reply:
Yes of course. But given that this is indeed directly related to the number of singly occupied sites, it might be worth mentioning the connection.